### PR TITLE
Add broker_settings.yaml to Dockerfile

### PR DESCRIPTION
**Pull request by JacobCallahan**
This should allow the build to complete.
Anyone using this container will need to overwrite settings at runtime.

### CODE REVIEW

**Review by mshriver — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -4,6 +4,7 @@ MAINTAINER https://github.com/SatelliteQE
 RUN ln -s /usr/bin/python3 /usr/bin/python
 RUN dnf install -y gcc git libffi-devel openssl-devel python3-devel \
     redhat-rpm-config which libcurl-devel libxml2-devel make
+RUN wget -o broker_settings.yaml https://raw.githubusercontent.com/SatelliteQE/broker/master/broker_settings.yaml.example
```
Comment (Gauravtalreja1) [inline Dockerfile]: @JacobCallahan use `-O` instead of `-o` with wget

**Review by Gauravtalreja1 — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -4,6 +4,7 @@ MAINTAINER https://github.com/SatelliteQE
 RUN ln -s /usr/bin/python3 /usr/bin/python
 RUN dnf install -y gcc git libffi-devel openssl-devel python3-devel \
     redhat-rpm-config which libcurl-devel libxml2-devel make
```
Comment (jameerpathan111) [inline Dockerfile:6]: there was an issue with pip not available, installing python3-pip solved that for me

**Review by jameerpathan111 — APPROVED** [review]
ACK

**Comment by jameerpathan111** [comment]
No idea why travis is taking so long to finish. @mshriver @JacobCallahan you guys can merge this if it's ok.

**Pull request merged**